input/pagecontent/index.md

```diff
@@ -11,6 +14,8 @@ The AU eRequesting project includes:
 - AU eRequesting Data for Interoperability
```

Comment:
Changes look good. 

Just a minor comment re existing Sparked AU FHIR Accelerator hyperlink resolving to the Confluence page, please update to go to https://sparked.csiro.au/.

Reply:
fixed + added care setting to scope sentence.